modules/cake.js

```diff
@@ -160,35 +159,17 @@ Module.addEvent("ready", () => {
     u.errorHandler(e, "Cakeday Init");
   }
 })
-// Janky stuff, but it works!!! (for now lol)
-.setUnload((date, type) => {
-  if (type === "cake") testCakeDays(undefined, date);
-  else if (type === "bday") testBirthdays(undefined, date);
-})
-.addCommand({ name: "bday",
-  permissions: () => config.devMode,
-  hidden: true,
-  process: (msg) => {
-    testBirthdays([{ discordId: msg.author.id, ign: new Date() }], new Date());
-  }
-})
-.addCommand({ name: "cakeday",
-  permissions: () => config.devMode,
-  hidden: true,
-  process: (msg, suffix) => {
-    const date = new Date();
-    if (suffix) date.setFullYear(parseInt(suffix));
-    testCakeDays(date, new Date(), new u.Collection().set(msg.author.id, msg.member));
-  }
-})
 .setClockwork(() => {
   return setInterval(() => {
     try {
-      celebrate();
+      doCelebrate();
     } catch (error) {
       u.errorHandler(error, "Birthday Clockwork Error");
     }
   }, 60 * 60 * 1000);
 });
 
 module.exports = Module;
+module.exports.doBirthdays = doBirthdays;
+module.exports.doCakeDays = doCakeDays;
+module.exports.doCelebrate = doCelebrate;
```

Comment:
This method doesn't work with the workspace version of typescript, at least in VSC. it might work, but it doesn't pass the ts check

Reply:
these lines don't show any errors on vsc on my computer, but also it does run just fine and by my estimation is less janky then hijacking the unload function. should I add an ignore flag for ts or leave as-is or do you think the unload function system is better?

Reply:
I think the `{ ...Module, doBirthdays, doCakedays, doCelebrate }` would work better. It actually shows up in imports when exported this way.